cli/commands/build.js

```diff
@@ -315,7 +317,7 @@ function patchLogger(logger, cli) {
 			// log it to our log file, stripping out the color codes
 			logger.log.filestream.write('\n' + prefix + (args.length > 2 ? sprintf.apply(null, args.slice(1)) : args[1]).replace(/\x1B\[\d+m/g, ''));
 		} else {
-			logger.log.buffer += '\n' + prefix + args[1].replace(/\x1B\[\d+m/g, '');
+			!logger.fileWriteEnabled && (logger.log.buffer += '\n' + prefix + args[1].replace(/\x1B\[\d+m/g, ''));
```

Comment:
You are trying to buffer the data only if the fileWriteEnabled is false. Why do you need to buffer the data if we close the file stream, but probably never open it again ?

Shouldn't the entire block be guarded against fileWriteEnabled condition ?